example_project_app/plugins.py

```diff
@@ -19,6 +19,44 @@
 )
 
 
+def get_example_setting_default(project=None, user=None):
+    """
+    Example callable function for different scopes
+
+    :param project: Project object
+    :param user: User object
+    :return: String with project and user info or 'No project'
+    """
+    response = 'No project or user for callable'
+    if project and user:
+        response = '{}:{}'.format(project.title, user.username)
+    elif project:
+        response = project.title
+    elif user:
+        response = user.username
+    return response
+
+
+def get_example_setting_options(project=None, user=None):
+    """
+    Example callable function for options forming
+
+    :param project: Project object
+    :param user: User object
+    :return: List of strings
+    """
+    response = ['No project or user for callable']
```

Comment:
I would like for the example method to return proper (value, label) tuples instead of a simple list, so it serves as a nice example for the potential users of the feature. You can e.g. use the UUID of the project/user as the actual value (with empty value for no project/user of course)

Reply:
I think here is problem in Github UI)) I've changed it to tuples, see the code section